app/views/UkimsNumberView.scala.html

```diff
@@ -27,19 +28,18 @@
     link: components.Link
 )
 
-@(form: Form[_])(implicit request: Request[_], messages: Messages)
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
 
 @layout(
     pageTitle    = titleNoForm(messages("ukimsNumber.title")),
-    showBackLink = true,
-    backLinkUrl = controllers.routes.ProfileSetupController.onPageLoad.url
+    showBackLink = true
```

Comment:
This will not work now since we delete the JS back button behaviour in a previous ticket https://github.com/hmrc/trader-goods-profiles-frontend/commit/d30688f1dfd44fcc3d4a5c8114ff9170537d15e2

@liandra-giacomine Shall I fix this in my ticket https://jira.tools.tax.service.gov.uk/browse/TGP-801 ?

Reply:
What's the plan for this? Am happy to add the change?

Reply:
Back links aren't being tested yet but I am pro having stuff that works - whether we want to add the JS button functionality back or add backLinkUrl = controllers.routes.ProfileSetupController.onPageLoad.url back in I don't mind so long as it isn't causing a failure, as this will be worked on in a future ticket

Reply:
@besscerule you can see the change here https://github.com/hmrc/trader-goods-profiles-frontend/commit/d30688f1dfd44fcc3d4a5c8114ff9170537d15e2

app.js 

`document.addEventListener('DOMContentLoaded', function(event) {

   // handle back click
   var backLink = document.querySelector('.govuk-back-link');
   if (backLink !== null) {
     backLink.addEventListener('click', function(e){
       e.preventDefault();
       e.stopPropagation();
       window.history.back();
     });
   }
 });`

Layout.scala.html

`@(pageTitle: String, showBackLink: Boolean = true, timeout: Boolean = true, showSignOut: Boolean = true)(contentBlock: Html)(implicit request: Request[_], messages: Messages)`

`@if(showBackLink) {
         @govukBackLink(BackLinkViewModel(href = "#").withCssClass("js-visible"))
     }`

Reply:
fixed